nectar/downloaders/local.py

```diff
@@ -158,7 +158,7 @@ def _copy(self, request, report=None):
                 last_progress_update = now
 
         except IOError, e:
-            logger.debug(e)
+            logger.info(e)
```

Comment:
From my testing, these exceptions usually display the path (e.g. [Errno 13] Permission denied: '///home/vagrant/zoolander/cat-1.0-1.noarch.rpm')